### PR TITLE
[Fix-7236][common] Fix bug of the host cannot be recognized when the endpoint uses the domain name

### PR DESCRIPTION
**Pull request by liqiquan**
…endpoint uses the domain name

fix bug：#7236

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@
         <spring.boot.version>2.5.6</spring.boot.version>
         <java.version>1.8</java.version>
         <logback.version>1.2.3</logback.version>
-        <hadoop.version>2.7.3</hadoop.version>
+        <hadoop.version>2.8.1</hadoop.version>
```
Comment (lenboo) [inline pom.xml]: Is it necessary to upgrade this hadoop version ?
To upgrade the jar package, you need to modify both license and note.

Comment (liqiquan) [reply pom.xml]: The problem is in the org.apache.hadoop:hadoop-ws jar.
Hadoop has officially solved this problem, but it has only been submitted to versions 2.8.x and 3.0.0-alpha1. This is the  [link](https://issues.apache.org/jira/browse/HADOOP-12963)

In order to solve this problem, our company has upgraded hadoop to 2.8.5。
I will modify both license and note later.

**Comment by caishunfeng** [comment]
Hi @liqiquan , please resolve the conflicts.

**Comment by liqiquan** [comment]
> Hi @liqiquan , please resolve the conflicts.

**Pull request closed without merging**